cmake/FindVCL.cmake

```diff
@@ -0,0 +1,27 @@
+# FindVCL
+#
+# Finds C++ vector class library, version 1
+# https://github.com/vectorclass/vesion1
```

Comment:
Could you change this to version 2, now that we moved to C++-17

Reply:
Fixed in https://github.com/feltor-dev/feltor/pull/52/commits/96e133a9ff84985d1c60adbba8c8caae164177aa